generate.2.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
-BASE=$(pwd)
-
-while getopts g:a:t:p:o:v:m:u: option
+while getopts g:a:t:p:o:v:m:u:w: option
```

Comment:
Also, I've not moved the file, i created its duplicate with some update in the root directory.